cms/envs/common.py

```diff
@@ -81,6 +81,8 @@
 
     # Enable or disable theming
     ENABLE_COMPREHENSIVE_THEMING,
+    COMPREHENSIVE_THEME_LOCALE_PATHS,
```

Comment:
Is it safe to switch this from an empty array initialization to an import?  Seems like there could be some environments where that'll change the current value.

Reply:
It was defined the exact same way in lms/envs/common.py:
https://github.com/edx/edx-platform/blob/master/lms/envs/common.py#L3234
So I assumed importing it along with these other defs was essentially equivalent - correct?

Reply:
I guess so; I was concerned about overrides of that value in the LMS settings, but if you're importing directly from the `common` module that shouldn't be an issue.